site/helm-charts.md

```diff
@@ -0,0 +1,513 @@
+# Using operator Helm Charts
+
+## Overview
+
+The WebLogic Kubernetes Operator uses Helm to create and deploy any necessary resources and then run the operator in a Kubernetes cluster. Helm helps you manage Kubernetes applications. Helm Charts help you define and install applications into the Kubernetes cluster. The operator's Helm Chart is located in the "kubernetes/charts/weblogic-operator" directory.
+
+## Install Helm and Tiller
+
+Helm has two parts: a client (helm) and a server (tiller). Tiller runs inside of your Kubernetes cluster, and manages releases (installations) of your charts.  See https://github.com/kubernetes/helm/blob/master/docs/install.md for detailed instructions on installing helm and tiller.
+
+## Operator's Helm Chart values file
+
+The operator Helm Chart comes with a built-in values.yaml that has all the default values for the configuration of the operator. This file is located in the "kubernetes/charts/weblogic-operator" directory. Users can create their own customized values.yaml file that can contain both additional and override property values for the operator. This is the recommended approach, rather than changing the supplied values.yaml file.
+
+The available values and their categories for the values.yaml file are explained in [Operator Helm Chart values.yaml details](#operator-helm-chart-values.yaml-details) below.
+
+## Optional: Configure the operator's external REST https interface
+
+The operator can expose an external REST https interface which can be accessed from outside the Kubernetes cluster. As with the Operator's internal REST interface, the external REST interface requires an SSL certificate and private key that the operator will use as the identity of the external REST interface (see below).
+
+To enable the external REST interface, configure these properties values.yaml file:
```

Comment:
configure these properties values.yaml file -> configure these properties in the values.yaml file

Reply:
Obsolete